src/main/java/org/mdbenefits/app/utils/ExpenseCalculator.java

```diff
@@ -28,9 +28,12 @@ public BigDecimal totalUtilitiesExpenses() {
         expenses.forEach(val -> {
             if (!val.equalsIgnoreCase("None")) {
                 String inputFieldName = HomeExpensesType.getEnumByName(val).getInputFieldName();
-                expenseAmounts.add(
-                        new BigDecimal(inputData.getOrDefault(inputFieldName, "0").toString())
-                                .setScale(2, RoundingMode.HALF_UP));
+                // expense values are not required and may be blank
+                String expenseValue = inputData.getOrDefault(inputFieldName, "0").toString();
+                if (expenseValue.isBlank()) {
+                    return;
```

Comment:
Should this not be `continue`? `return` will stop the loop all together and return control the caller right? Where `continue` will stop the current iteration and move to the next?

Reply:
I think that because it's an anonymous function, that we need `return` versus `continue`, since the function is called once per entry, rather than it being a standard loop.  I'll confirm this though.

Reply:
Lambda's need the `return`. Essentially a `return` equals `break` in this context.  The code won't even compile if I use `break` there, which makes sense as it's not really in a for loop, but rather a function call.  (if you think of the thing that calls this lambda as the "for loop" this makes a lot of sense).

Reply:
That makes sense, in that case this looks good to go!